### ISSUE TITLE
argument type of id fields seems to change in autogenerated schema

### ISSUE THREAD

**Issue by shrugs** [0xOlias/ponder#251]
`@ponder/core@0.0.70`

Token is an entity with `id: BigInt!` here:

```gql
 query TokenInfo($id: ID!) {
    token(id: $id) { # demands ID!
      id
      multiplier
    }
    # the mint event
    transferEvents(
      where: { from: "0x0000000000000000000000000000000000000000", token: $id } # demands String!
    ) {
      id
      to {
        id
      }
      transactionHash
      timestamp
    }
  }
```

returns

```json
{
  "errors": [
    {
      "message": "Variable \"$id\" of type \"ID!\" used in position expecting type \"String\".",
      "locations": [
        {
          "line": 1,
          "column": 18
        },
        {
          "line": 8,
          "column": 75
        }
      ]
    }
  ]
}
```

**Comment by shrugs** [comment]
this may or may not be a bug, depending on perspective, but it's definitely an ergonomics issue that doesn't exist with `String` ids

**Comment by shrugs** [comment]
ah, this does occur with a `String!` id now, seems to be a change introduced in `0.0.70`

**Comment by 0xOlias** [comment]
Should be fixed in `0.0.71`
